lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -116,11 +116,20 @@
             },
 
             render: function () {
+                var linkTitle;
+                if (this.options.connected) {
+                    linkTitle = gettext('Unlink');
+                } else if (this.options.acceptsLogins) {
+                    linkTitle = gettext('Link')
+                } else {
+                    linkTitle = ''
+                }
```

Comment:
So, to clarify, this is the page where one can unlink an existing social auth provider, or link a new one. Without this change, unlinking an LTI provider would cause the item to vanish, rather than simply change to Unlink in the case of any other type of provider. I made this change to make the interaction more uniform. Note that the provider does not appear after a page refresh or if it was never linked at all.